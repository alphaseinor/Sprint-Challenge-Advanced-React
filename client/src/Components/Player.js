import React from 'react'

class Player extends React.Component {
  render(){
  return(
    <article key={this.props.id}>
      <h3>{this.props.player.name}</h3>
      <p>{this.props.player.country}</p>
    </article>
  )}
}

export default Player;