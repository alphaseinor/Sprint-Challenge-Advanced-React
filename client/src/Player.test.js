import React from 'react'
import { render, cleanup } from '@testing-library/react'
import '@testing-library/jest-dom'
import Player from './Components/Player'

afterEach(cleanup);

const data = {
  name: "brian",
  country: "brianlandia"
}

it('renders', () => {
  const {asFragment} = render(<Player key={1} player={data} />)
  expect(asFragment()).toMatchSnapshot();
});
