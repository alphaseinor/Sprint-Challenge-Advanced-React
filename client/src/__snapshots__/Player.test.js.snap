// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <article>
    <h3>
      brian
    </h3>
    <p>
      brianlandia
    </p>
  </article>
</DocumentFragment>
`;
